BraveWallet/WalletStrings.swift

```diff
@@ -0,0 +1,1398 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import struct Shared.Strings
+
+extension Strings {
+  struct Wallet {
+    public static let portfolioPageTitle = NSLocalizedString(
+      "wallet.portfolioPageTitle",
```

Comment:
not looking for duplicates here, keep an eye on them when exporting to transifex, all duplicates should show up in output.log after it's exported